CHANGELOG.md

```diff
@@ -79,7 +79,7 @@ stage in the `PlannerPipeline` and to generate performance metrics for the indiv
 
 ### Removed
 - README.md badge for travisci
-
+- The following custom type AST nodes are removed from `partiql.ion` file: `es_boolean`, `es_integer`, `es_float`, `es_text`, `es_any`, `spark_short`, `spark_integer`, `spark_long`, `spark_double`, `spark_boolean`, `spark_float`, `rs_varchar_max`, `rs_integer`, `rs_bigint`, `rs_boolean`, `rs_real`, `rs_double_precision`. The related visitor transform `CustomTypeVisitorTransform` is also removed
```

Comment:
Would it make sense to also document rationale and/or consequences for this change?  I did not yet reverse-engineer what they are, but I imagine something like 
"PartiQL did not have custom support for these types, beyond parsing, and we are not aware of currently-active users.  The effect of these nodes can be achieved with the generic (custom_type ...) node".

Reply:
I don't know much about the story behind, but I guess that originally, we had these specific custom type AST nodes. Later, we decided to wrap all of them using one generic custom type AST node `custom_type`, and at that moment, we just forgot (or not confident enough) to remove these specific AST nodes. At least, from what I can see, these specific AST nodes are not really useful and they are not used anyway with regard to the current custom type API in our codebase. So I think we can remove them safely.

Reply:
I think we should only document the changes in short in the `CHANGELOG.md` file. I also wonder if we should document rationale behind these changes somewhere else.

Reply:
I see fairly detailed entries in the current [Unreleased] section.  It does look somewhat of an outlier compared to the earlier, but is that history long / active enough to take as the precedent law?
At the very least, a reference to Issue 510 + adding this explanation there, I think.

Reply:
+1 to what Vladimir had mentioned. At the very least would be helpful to add a link to issue #510's comment